wagtail/admin/tests/viewsets/test_model_viewset.py

```diff
@@ -452,6 +453,44 @@ def assertInputRendered(self, response, search_q):
         pass
 
 
+class TestSearchFields(WagtailTestUtils, TestCase):
```

Comment:
This issue is in the generic IndexView (not specific to snippets), so I opted to move the test to `test_model_viewset` (and rename the model to not have "snippet" in its name).